### PR TITLE
Death squad amped

### PR DESCRIPTION
**Pull request by Looking4bros**
## About The Pull Request

Adds to and updates the capabilities of Deathsquad members.

- Upgrades night vision goggles to current in game best
- Gives Deathsquad armor the grenade storage module, and packs the additional storage with specialty grenades.
- Updates the Medical kit and Elite Injector, ~~now about as strong as i can make it without using Adminorazine, or pushing the kidneys past their organ threshold. (Considered nerfing Ultrazine and putting it in as well, but seems like that'd be removing the primary weakness of Deathsquad.~~ (Also nobody needs the kits, you should be done killing everyone by the time kits would come into play.)
- Adds Binoculars, because you may as well scope out the place if you're going to fat run everywhere.
- Gives the leader a Xeno Structure pinpointer, just in case.
- Made Energy swords and shields strapable, and buffs the Deathsquad's sword specifically to be a little better than SOMs. (was weaker)
- Helmet now has eye protection and flashbang immunity, so when they get blown up (and take zero damage) they don't have to worry about having a white screen.
- Updates the smartgunners magazine storage pouch and re implements it, fits four drums to match the Leaders explosive pouch holding four four-packs of rockets.
- Changes CL's thingamabober because _technically_ they could have gotten an elite autoinjector beforehand.

## Why It's Good For The Game

Sometimes people just need to die. This will help.

## Changelog
:cl:
balance: More Deathsquad updating.
balance: Energy sword (which is issued to SOM as well) is now strapable. (shield is not issued to anyone else)
/:cl:

### CODE REVIEW

**Comment by Looking4bros** [comment]
Need to make a custom Toxin heal pillbottle and replace the Rye packet, but words and design elude me. Runs into the issue of potential OD and Kidney overload.
Also, Why linters fail!! why is!!

**Comment by Looking4bros** [comment]
tested how i made the helmet flashbang immune, works for those wearing it, and not for everyone around them. hashtag swag.

**Comment by AnotherNobodie** [comment]
For the toxin heal couldn't you make a hyro rye pill bottle?

**Comment by Looking4bros** [comment]
yeah, it's more so just making the pill bottle and pills not ugly. I tried my hand at it and they look gross. i actually like the look of the packet, maybe i'll do that.

File: code/game/objects/items/reagent_containers/autoinjectors.dm [hunk 1]
```diff
@@ -297,9 +297,19 @@
 	name = "elite autoinjector"
 	desc = "An elite auto-injector loaded with a strong and versatile combination of chemicals, healing most types of damage. Issued almost excusively to the infamous Nanotrasen deathsquads"
 	icon_state = "autoinjector-7"
+	volume = 60
+	amount_per_transfer_from_this = 30
 	list_reagents = list(
```
Comment (ivanmixo) [inline code/game/objects/items/reagent_containers/autoinjectors.dm:302]: Errr what the hell? Tea? Can you just keep it to rezadone and RR please?

**Review by ivanmixo — CHANGES_REQUESTED** [review]

File: code/game/objects/items/reagent_containers/autoinjectors.dm (hunk 1 above)
Comment (Looking4bros) [reply code/game/objects/items/reagent_containers/autoinjectors.dm:302]: it's so joever...

**Review by ivanmixo — APPROVED** [review]

**Pull request merged**